doc/man/isympy.1

```diff
@@ -169,14 +169,14 @@ from the other isympy options.
 For example, to run iSymPy without startup banner and colors:
 
 isympy -q -c ipython -- --colors=NoColor
-.TP 
+.TP
 \*(T<\fB\-h\fR\*(T>, \*(T<\fB\-\-help\fR\*(T>
 Print help output and exit.
-.TP 
+.TP
 \*(T<\fB\-v\fR\*(T>, \*(T<\fB\-\-version\fR\*(T>
 Print isympy version information and exit.
 .SH FILES
-.TP 
+.TP
```

Comment:
Is trailing whitespace significant in troff files?

Reply:
Good point. I assume(d) that it would not have an effect, but it's maybe better to ask: https://stackoverflow.com/q/61487473/562769 (and check; but I don't know how)